src/interaction/command/admin.ts

```diff
@@ -9,13 +9,7 @@ export default class AdminCommand extends SlashCommand {
     super({
       name: "admin",
       description: "Staff commands",
-      permissions: [
-        {
-          id: "312383932870033408",
-          type: "USER",
-          permission: true,
-        },
-      ],
+      permissions: 0,
```

Comment:
I think theres a bug here, where you just check `if permissions` when building slash/cm commands. This returns false for 0 and (i think) leaves it enabled for everyone by default. Should instead check `if permissions !== undefined` probably..

Reply:
Not super important because we can override the default perms in server settings